extensions/packageManager/src/client/Library.tsx

```diff
@@ -146,11 +153,12 @@ const Library: React.FC = () => {
     // find all categories listed in the available libraries
     const categories = [DEFAULT_CATEGORY];
     if (availableLibraries) {
-      const availableCompatibleLibraries = availableLibraries.filter(component => isCompatible(component));
+      const availableCompatibleLibraries = availableLibraries; // .filter(component => isCompatible(component));
```

Comment:
can the comment be removed? I see `isCompatible` was added to the item below.

Reply:
yup thanks!